frontend/src/layout/Sidebar.js

```diff
@@ -240,6 +244,15 @@ export function Sidebar({ user, sidebarCollapsed, setSidebarCollapsed }) {
                             <span className="sidebar-label">{'Annotations'}</span>
                             <Link to={'/annotations'} onClick={collapseSidebar} />
                         </Menu.Item>
+
+                        {featureFlags && featureFlags['billing-management-page'] && (
```

Comment:
This is all great! Nicely done!
The only nit I can give is that `featureFlags` is always defined, so no need for the `&&`. Definitely not worth opening another PR for that

Reply:
Oh you're right, thanks for pointing that out, I'll remember to fix it in my next frontend PR.